CONTRIBUTING.md

```diff
@@ -39,16 +39,24 @@ This will take ~ 2 hours, YMMV... you can up the jobs to 4 and run in parallel,
 $ cd build/cppcache/integration-test
 $ ctest -R <test_name> -C <Debug|Release>
 ```
+For example: `$ ctest --timeout 2000 -L STABLE -C Release -R testCacheless -j1`
+
 .NET integration tests can be executed similarly from `build/clicache/integration-test`.
 
-### Running new integration tests
+#### Running Google Test integration test suite
 ```bash
 $ cd <clone>
-$ cd build
-$ cd cppcache/integration/test
-$ ./cpp-integration-test [<options>]
+$ cd build/cppcache/integration/test
+$ ctest -R . -j1
```

Comment:
Depending on your host, you should be safe to run a few in parallel. Also `-R .` is redundant to no `-R` argument at all. I am not sure where the `-R .` stated sneaking into this document.